src/ExternalImage.php

```diff
@@ -0,0 +1,479 @@
+<?php
+
+namespace Timber;
+
+/**
+ * Class ExternalImage
+ *
+ * The `Timber\ExternalImage` class represents an external image loaded via http.
```

Comment:
With this description and the name `ExternalImage`, I would assume that I could only use image URLs starting with `http(s)://` and that I could either use a path from my own site or load an image from a completely different domain.

The code in the build function says otherwise. Apparently I can load images:

- from URLs from the same website
- from absolute paths
- from relative paths

I think we should reflect this in the documentation.

I think this is the reason why I’m still not sure about the name `External`. External in this case would mean "outside of the WordPress attachment area, but still on the same site". But don’t we often use the term "external" to stuff from other websites?

To download external images, we already have a `ImageHelper::sideload_image()` that could be used in combination with `Timber::get_external_image()`. If we can also download external images, then I think I would be okay with the name `ExternalImage`.

Reply:
Yeah, external may not be the best name... And, actually, it doesn't really work 100% with external images.

I didn't know about `sideload`, but I'll integrate that, so external URLs also work

So if it works for:
- relative paths
- absolute paths
- arbitrary URLs (either same website, or different website)
would you feel more confortable with the `External` name?

Reply:
Yes, if we support all of those, then I’d be okay with the name `External` 👍